app.yaml

```diff
@@ -3,5 +3,6 @@ runtime: nodejs10
 resources:
  cpu: 1
  memory_gb: 0.5
- disk_size_gb: 11
+ disk_size_gb: 100
```

Comment:
This seems like a hefty increase, should we add a comment for why it's necessary?

Reply:
Oh I changed it back to 11. I changed this because I kept getting a space issue, but that was because of an issue in my terminal not appengine.